### PR TITLE
[PyOV][Docs] Add docs for if-op

### PR DESCRIPTION
**Pull request by akuporos**
### Details:
 - Added docs for `if` operation
 - fix typos

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: src/bindings/python/src/pyopenvino/graph/ops/if.cpp [hunk 1]
```diff
@@ -29,34 +39,155 @@ void regclass_graph_op_If(py::module m) {
                     return std::make_shared<ov::op::v8::If>();
                 }
             }),
-            py::arg("execution_condition"));
-    cls.def("get_else_body", &ov::op::v8::If::get_else_body);
-    cls.def("set_then_body", &ov::op::v8::If::set_then_body, py::arg("body"));
-    cls.def("set_else_body", &ov::op::v8::If::set_else_body, py::arg("body"));
+            py::arg("execution_condition"),
+            R"(
+            Constructs If with condition
```
Comment (jiwaszki) [inline src/bindings/python/src/pyopenvino/graph/ops/if.cpp:44]:
```suggestion
            Constructs If with condition.
```

File: src/bindings/python/src/pyopenvino/graph/ops/if.cpp [hunk 2]
```diff
@@ -19,7 +19,17 @@ void regclass_graph_op_If(py::module m) {
     py::class_<ov::op::v8::If, std::shared_ptr<ov::op::v8::If>, ov::Node> cls(m, "if_op");
     cls.doc() = "openvino.impl.op.If wraps ov::op::v0::If";
     cls.def(py::init<>());
-    cls.def(py::init<const ov::Output<ov::Node>&>(), py::arg("execution_condition"));
+    cls.def(py::init<const ov::Output<ov::Node>&>(),
+            py::arg("execution_condition"),
+            R"(
+            Constructs If with condition
```
Comment (jiwaszki) [inline src/bindings/python/src/pyopenvino/graph/ops/if.cpp:25]:
```suggestion
            Constructs If with condition.
```

File: src/bindings/python/src/pyopenvino/graph/ops/if.cpp [hunk 3]
```diff
@@ -29,34 +39,155 @@ void regclass_graph_op_If(py::module m) {
...
             &ov::op::v8::If::set_input,
             py::arg("value"),
             py::arg("then_parameter"),
-            py::arg("else_parameter"));
-    cls.def("set_output", &ov::op::v8::If::set_output, py::arg("then_result"), py::arg("else_result"));
-    cls.def("get_function", &ov::op::util::MultiSubGraphOp::get_function, py::arg("index"));
-    cls.def("set_function", &ov::op::util::MultiSubGraphOp::set_function, py::arg("index"), py::arg("func"));
+            py::arg("else_parameter"),
+            R"(
+            Sets new input to the operation associated with parameters of each sub-graphs
```
Comment (jiwaszki) [inline src/bindings/python/src/pyopenvino/graph/ops/if.cpp:91]:
```suggestion
            Sets new input to the operation associated with parameters of each sub-graphs.
```

File: src/bindings/python/src/pyopenvino/graph/ops/if.cpp [hunk 4]
```diff
@@ -29,34 +39,155 @@ void regclass_graph_op_If(py::module m) {
...
         [](const std::shared_ptr<ov::op::v8::If>& self, int index, const py::list& inputs) {
             self->set_input_descriptions(index, MultiSubgraphHelpers::list_to_input_descriptor(inputs));
         },
         py::arg("index"),
-        py::arg("inputs"));
+        py::arg("inputs"),
+        R"(
+            Sets list with connections between operation inputs and internal sub-graph parameters
+
+            :param index: index of internal sub-graph
```
Comment (jiwaszki) [inline src/bindings/python/src/pyopenvino/graph/ops/if.cpp:161]:
```suggestion
            :param index: index of internal sub-graph.
```

**Review by jiwaszki — COMMENTED** [review]
There is an inconsistent style of either using or omitting dots at the end of sentences. Besides that LGTM.

File: src/bindings/python/src/pyopenvino/graph/ops/if.cpp (hunk 4 above)
Comment (akuporos) [reply src/bindings/python/src/pyopenvino/graph/ops/if.cpp]: added

File: src/bindings/python/src/pyopenvino/graph/ops/if.cpp (hunk 3 above)
Comment (akuporos) [reply src/bindings/python/src/pyopenvino/graph/ops/if.cpp]: added

File: src/bindings/python/src/pyopenvino/graph/ops/if.cpp (hunk 2 above)
Comment (akuporos) [reply src/bindings/python/src/pyopenvino/graph/ops/if.cpp]: added

File: src/bindings/python/src/pyopenvino/graph/ops/if.cpp (hunk 1 above)
Comment (akuporos) [reply src/bindings/python/src/pyopenvino/graph/ops/if.cpp]: added

**Review by jiwaszki — APPROVED** [review]

**Review by p-wysocki — APPROVED** [review]

**Pull request merged**